tests/unit/test_tf_dataloader.py

```diff
@@ -33,6 +33,55 @@
 tf_dataloader = pytest.importorskip("nvtabular.loader.tensorflow")
 
 
+@pytest.mark.parametrize("batch_size", [10, 9, 8])
+@pytest.mark.parametrize("drop_last", [True, False])
+@pytest.mark.parametrize("num_rows", [100])
+def test_tf_drp_reset(tmpdir, batch_size, drop_last, num_rows):
```

Comment:
Should we test the Torch side too? Wondering how different that test would be. If it's mostly the same, maybe the overlapping parts could be a `DataLoader` test?

Reply:
This is a good idea... do we want to work on that here, or in another PR?

Reply:
Based on your comment above re: Torch, it seems like Torch doesn't have any particular code to test. I could go either way on pursuing a `DataLoader` test here or in a subsequent PR.

Reply:
So for sure I am working on the test for the torch side... because it should be there. But I want to test the actual framework specific code also, So I don't know if this should be a DataLoader level test. But we should have DataLoader level tests for sure.

Reply:
Yeah, testing it for both frameworks covers the bases nicely. :+1: